public/index.html

```diff
@@ -10,5 +10,5 @@
 <body>
     <div id="app"></div>
     <script src="./bundle.js"></script>
-</body>
+<script type="text/javascript" src="/bundle.js"></script><script type="text/javascript" src="/bundle.js"></script><script type="text/javascript" src="/bundle.js"></script><script type="text/javascript" src="/bundle.js"></script></body>
```

Comment:
What is this for? @McHardex

Reply:
Fixed this. Came from my initial html webpack plugin setup